cmd/svcat/README.md

```diff
@@ -0,0 +1,211 @@
+# Service Catalog CLI
+
+This is a command-line interface (CLI) for interacting with
+[Kubernetes Service Catalog](../../README.md)
+resources. svcat is a domain-specific tool to make interacting with the Service Catalog easier.
+While many of its commands have analogs to `kubectl`, our goal is to streamline and optimize
+the operator experience, contributing useful code back upstream to Kubernetes where applicable.
+
+svcat communicates with the Service Catalog API through the [aggregated API][agg-api] endpoint on a
+Kubernetes cluster.
+
+[agg-api]: https://kubernetes.io/docs/concepts/api-extension/apiserver-aggregation/
+
+# Prerequisites
+In order to use svcat, you will need:
+
+* A Kubernetes cluster running v1.7+ or higher
+* A broker compatible with the Open Service Broker API installed on the cluster, such as:
+  * [User Provided Service Broker](../../charts/ups-broker)
+  * [Open Service Broke for Azure](https://github.com/Azure/helm-charts/tree/master/open-service-broker-azure)
+* The [Service Catalog](../../docs/install.md) installed on the cluster.
+
+# Install
+Follow the appropriate instructions for your shell to download svcat. The binary
+can be used by itself, or as kubectl plugin.
+
+## Bash
+```
+curl -sLO https://servicecatalogcli.blob.core.windows.net/cli/latest/$(uname -s)/$(uname -m)/svcat
```

Comment:
Will the built versions continue to be built and hosted by this URI after the tool have been moved into service-catalog?

Reply:
This is a very good point.

Reply:
I mentioned in the OP that these would need to be changed once we start building and publishing binaries during the build.